app/controllers/users/registrations_controller.rb

```diff
@@ -10,13 +10,10 @@ def respond_with(resource, _opts = {})
   end
 
   def register_success
-    # message = 'Signed in successfully ${@user.user_name}'
-    # render json: { message => :message }
-    # render json: { message: 'Signed up sucessfully.' }
```

Comment:
I would suggest you remove some commented code in your code base, as this will make code base looks cleaner.